### DISCUSSION TITLE
!me section missing from profiles.

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#17027] [Q&A] [answered]
So something I have noticed for a long time now but hasn't been added yet is the !me or bio section of profiles. Is there a specific reasoning behind this or is it just something that is a low priority?

**Comment by zachmanthethird** [comment]
I can't answer certainly; however, it probably will be implemented with the new designs.
See https://www.figma.com/file/ocltATjJqWQZBravhPuqjB/UI%2FPlayer-Card?node-id=0%3A1

**Reply by frenzibyte** [reply]
The designs are irrelevant, the me! section already exists in the current design osu!web-side. I'm not even sure where that is in the linked figma design.

**Comment by frenzibyte** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/8022

**Reply by TheFantasticLoki** [reply]
So the only issue is that BBcode is not implemented then? Is this under such careful consideration due to potential for exploitation or something else?

**Reply by peppy** [reply]
It just hasn't been done yet. I'm not sure what "exploitation" you refer to, but the complexity is in getting it rendering in a format we can use. See markdig/markdown implementation for an example of what needs to be done (assuming we are handling bbcode at the game, rather than performing conversion at the web end).

**Reply by TheFantasticLoki** [reply]
I wasn't referring to any specific exploit but anytime you add user input into a game it's usually best practice to keep an open mind and expect that some creative soul will find a way to exploit the system. I mean look at the log4j vulnerability that was discovered a little while ago. That's been present but undiscovered for years. 

And thank you for explaining that it's simply not implemented and giving an example to reference.

**Reply by TheFantasticLoki** [reply]
@peppy From a quick google search I was able to find this post about exploiting XSS in certain BBCode implementations.
Might be worth checking out as a just in case measure: https://jeffchannell.com/Other/bbcode-xss-howto.html

**Reply by peppy** [reply]
Please. Stop posting so much.
